### ISSUE TITLE
Disassociate capsules from lifecycle environments fails with db error

### ISSUE THREAD

**Issue by anandkumaragrawal** [RedHatSatellite/satellite-clone#376]
satellite-clone fails at disassociate capsule step with below error:
~~~
2020-11-10 11:02:49,593 p=48208 u=root |  fatal: [localhost]: FAILED! => {"changed": true, "cmd": ["psql", "foreman", "-c", "delete from katello_capsule_lifecycle_environments where capsule_id in (select smart_proxy_id from features_smart_proxies where feature_id = (select id from features where features.name = 'Pulp Node'));"], "delta": "0:00:00.012455", "end": "2020-11-10 11:02:49.575174", "msg": "non-zero return code", "rc": 1, "start": "2020-11-10 11:02:49.562719", "stderr": "ERROR:  relation \"features_smart_proxies\" does not exist\nLINE 1: ...s where capsule_id in (select smart_proxy_id from features_s...\n                                                             ^", "stderr_lines": ["ERROR:  relation \"features_smart_proxies\" does not exist", "LINE 1: ...s where capsule_id in (select smart_proxy_id from features_s...", "     
~~~

**Comment by chris1984** [comment]
Starting to review

**Comment by evgeni** [comment]
this has been fixed since 72940e19bf8251110f62e0c89d947da3d4990c68 and cleaned up in ee21516626675e92e59712350ed054f926475c79
